.github/workflows/main.yml

```diff
@@ -18,7 +18,15 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
           pip install pylint diff_cover
+          curl https://github.com/ydirson/branding-xcp-ng/releases/download/untagged-630bba873b965a46c408/branding.py -O
```

Comment:
We've discussed that using this for CI purposes would be fine (it needs a branding module, and we don't have one in this repository to avoid accidentally using the wrong branding).
This URL is a 404 now though, can you make it point to a more stable place?